ci/py311_simplification-latest.yaml

```diff
@@ -0,0 +1,43 @@
+name: py311_simplification-latest
+channels:
+  - conda-forge
+  - nodefaults
```

Comment:
what is this?

Reply:
Something that carried over from [`environment.yml`](https://github.com/uscuni/simplification/blob/ed070520a695e50a909906d13bf9a367d23470fd/environment.yml#L4). If I am not mistaken it was declared in @anastassiavybornova's original environment. Shall we try without?

Reply:
yes i think that can be dropped. i don't remember what this was for tbh (some stackexchange solution for some dependency issues i had?) and i think martin you already asked "what is this" more than once :D

Reply:
OK. I'll update this PR + `environment.yml` and repush.